### PR TITLE
Nerlnetplanner

### PR DESCRIPTION
**Pull request by ohad123**

### CODE REVIEW

File: src_py/nerlPlanner/Definitions.py [hunk 1]
```diff
@@ -23,6 +23,9 @@
 KEY_SETTINGS_APISERVER_PORT_INPUT = '-KEY-SETTINGS-APISERVER-PORT-INPUT-'
 KEY_SETTINGS_APISERVER_ARGS_INPUT = '-KEY-SETTINGS-APISERVER-ARGS-INPUT-' 
 KEY_SETTINGS_SAVE_BUTTON = '-KEY-SETTINGS-SAVE-BUTTON-'
+KEY_SETTINGS_CLEAR_BUTTON = '-KEY-SETTINGS-CLEAR-BUTTON-'
+KEY_SETTINGS_MAIN_SERVER_STATUS_BAR = '-KEY-SETTINGS-MAIN-SERVER-STATUS-BAR-'
```
Comment (leondavi) [inline src_py/nerlPlanner/Definitions.py:27]: Wrong typo

File: src_py/nerlPlanner/Handlers.py [hunk 2]
```diff
@@ -16,6 +16,16 @@ def reset_json_distributed_configuration():
     global json_dc_inst
     json_dc_inst = JsonDistributedConfig()
 
+def settings_reset_inputs_ui(window):
+    window[KEY_SETTINGS_FREQUENCY_INPUT].update('')
+    window[KEY_SETTINGS_BATCH_SIZE_INPUT].update('')
+    
+def settings_special_entities_reset_inputs_ui(window):
+    window[KEY_SETTINGS_MAINSERVER_PORT_INPUT].update('')
+    window[KEY_SETTINGS_MAINSERVER_ARGS_INPUT].update('')
```
Comment (leondavi) [inline src_py/nerlPlanner/Handlers.py:25]: Fix this line

**Review by leondavi — CHANGES_REQUESTED** [review]

File: src_py/nerlPlanner/Definitions.py (hunk 1 above)
Comment (ohad123) [reply src_py/nerlPlanner/Definitions.py:27]: done

File: src_py/nerlPlanner/Handlers.py (hunk 2 above)
Comment (ohad123) [reply src_py/nerlPlanner/Handlers.py:25]: done

**Review by leondavi — APPROVED** [review]

**Pull request merged**